src/main/java/view_user_public_profile_screens/ViewUserPublicProfilePresenter.java

```diff
@@ -0,0 +1,16 @@
+package view_user_public_profile_screens;
+import view_user_public_profile_usecase.ViewUserPublicProfileOutputBoundary;
+import view_user_public_profile_usecase.ViewUserPublicProfileResponseModel;
+
+
+public class ViewUserPublicProfilePresenter implements ViewUserPublicProfileOutputBoundary {
+    @Override
+    public ViewUserPublicProfileResponseModel prepareSuccessView(ViewUserPublicProfileResponseModel userPublicProfileInfo) {
+        return userPublicProfileInfo;
+    }
+
+    @Override
+    public String prepareFailView(String message) {
+        return null;
+    }
```

Comment:
Like mentioned before, consider making a screen specifically for fail (refer to Paul's log in example), that way you can throw the screen with the error.